ripples/multi_session.py

```
@@ -606,193 +1320,411 @@ def plot_ripple_triggered_firing(
 
-    mean_firing_around_ripples = np.empty((len(all_ripple_times), 200))
-    for ripple in range(len(all_ripple_times)):
-        ripple_peak = all_ripple_times[ripple].astype(float)
-        mean_firing_around_ripples[ripple] = count_spikes_around_ripple_simple(
-            ripple_peak,
-            spike_times,
-            padding=2,
-            num_bins=200,
-        )
+    for cluster_idx, cluster in enumerate(session.clusters_info):
+        if cluster.good_cluster != True or cluster.region != "CA1":
```

Ruff (0.8.2)

Avoid inequality comparisons to `True`; use `if not cluster.good_cluster:` for false checks

Replace with `not cluster.good_cluster`

(E712)